bidwire/notifiers/notifier_utils.py

```diff
@@ -80,4 +80,8 @@ def make_doc_item_body(document):
     with tag('strong'):
         with tag('a', href=document.url):
             text(document.title)
-    return doc.getvalue()
+    if document.description:
+        with tag('p'):
+            # it's already in HTML
+            doc.asis(document.description)
```

Comment:
If i'm reading your code correctly, this is not, actually, HTML (on line 29 of the scraper you are calling `.text`). And it shouldn't be HTML for storing in the database, anyway. So remove the comment, and make this `text(document.description)`.